.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: ./bin/run_docker_dev.sh test --no-flake8 --no-pytest
   static-analysis:
     name: Static Analysis
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Did you try just updating this to a newer Python version, rather than pinning the Ubuntu version?